### PR TITLE
[Feature/playbook] - Introduce apt-pkg-install role

### PR DESCRIPTION
**Pull request by yqlbu**
## Summary

Introduce a role tailored to installing a custom list of apt packages (works with DEBIAN based system)

## Notes

Define the list of software packages you wish to be installed on the target remote servers in the playbook with the `vars` attribute as shown in the following:

```yaml
---
- name: Install packages with apt
  hosts: all
  become: yes

  vars:
    - packages: # Packages to be installed
      - vim
      - neofetch
  roles:
    - role: ./roles/apt.ops/install-packages.ops/
```

To run the playbook:

```bash
ansible-playbook -K -i <your inventory list> $HOME/workspace/cloud-estate/playbooks/install-apt-packages.yml
```

Sample execution logs:

<details><summary>ansible-play.logs</summary>

</br>

```
PLAY [Install packages with apt] ***********************************************

TASK [Gathering Facts] *********************************************************
ok: [test-apt]

TASK [./roles/apt.ops/install-packages.ops/ : include_tasks] *******************
included: /home/kev/workspace/cloud-estate/playbooks/roles/apt.ops/install-packages.ops/tasks/info.yml for test-apt

TASK [./roles/apt.ops/install-packages.ops/ : (Pre-ops) Print the desired list of pkgs to be installed] ***
ok: [test-apt] =>
  msg: |-
    ----------------------------------------------
    The following packages will be installed:

    vim
    neofetch
    ----------------------------------------------

TASK [./roles/apt.ops/install-packages.ops/ : include_tasks] *******************
included: /home/kev/workspace/cloud-estate/playbooks/roles/apt.ops/install-packages.ops/tasks/install.yml for test-apt

TASK [./roles/apt.ops/install-packages.ops/ : Update package source list] ******
ok: [test-apt]

TASK [./roles/apt.ops/install-packages.ops/ : Install packages defined in the list] ***
ok: [test-apt]

TASK [./roles/apt.ops/install-packages.ops/ : Remove useless packages from the cache] ***
ok: [test-apt]

TASK [./roles/apt.ops/install-packages.ops/ : Remove dependencies that are no longer required] ***
ok: [test-apt]

TASK [./roles/apt.ops/install-packages.ops/ : include_tasks] *******************
included: /home/kev/workspace/cloud-estate/playbooks/roles/apt.ops/install-packages.ops/tasks/verify.yml for test-apt

TASK [./roles/apt.ops/install-packages.ops/ : Gather package facts] ************
ok: [test-apt]

TASK [./roles/apt.ops/install-packages.ops/ : Check if the package is installed] ***
skipping: [test-apt] => (item=vim)
skipping: [test-apt] => (item=neofetch)

TASK [./roles/apt.ops/install-packages.ops/ : (Post-ops) Print software versions] ***
ok: [test-apt] => (item=vim) =>
  msg: 'version: 2:8.2.2434-3ubuntu3.2'
ok: [test-apt] => (item=neofetch) =>
  msg: 'version: 7.1.0-3'

PLAY RECAP *********************************************************************
test-apt                   : ok=11   changed=0    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0
```

</details>

### CODE REVIEW

File: playbooks/install-apt-packages.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+- name: Install packages with apt
+  hosts: all
+  become: yes
+
+  vars:
+    - packages: # Packages to be installed
+      - vim
```
Comment (mptm436) [inline playbooks/install-apt-packages.yml:8]: what if I need to specify a version of that package to be installed

**Review by mptm436 — COMMENTED** [review]
nice work to split ops into three parts, beautiful

File: playbooks/install-apt-packages.yml (hunk 1 above)
Comment (yqlbu) [reply playbooks/install-apt-packages.yml:8]: Nice feature request. I will take that into a consideration.

Comment (yqlbu) [reply playbooks/install-apt-packages.yml:10]: Introduced the requested feature, please find the commit below ([681d51d](https://github.com/TechProber/cloud-estate/pull/88/commits/681d51d4814019a8128e496c3cb40614df3b89e0))

Comment (mptm436) [reply playbooks/install-apt-packages.yml:10]: cool

**Review by mptm436 — APPROVED** [review]

**Pull request merged**